src/libs/GrowlNotification.js

```diff
@@ -0,0 +1,101 @@
+import React, {
+    forwardRef, useImperativeHandle, useRef, useState,
+} from 'react';
+import {
+    Text, View, Animated, Platform,
+} from 'react-native';
+import {
+    Directions, FlingGestureHandler, State, TouchableWithoutFeedback,
+} from 'react-native-gesture-handler';
+import colors from '../styles/colors';
+import Icon from '../components/Icon';
+import {Checkmark, Exclamation} from '../components/Icon/Expensicons';
+import ScreenWrapper from '../components/ScreenWrapper';
+import styles from '../styles/styles';
+import withWindowDimensions, {windowDimensionsPropTypes} from '../components/withWindowDimensions';
+
+const types = {
+    success: {
+        icon: Checkmark,
+        iconColor: colors.green,
```

Comment:
@shawnborton I think we should be using theme colors, not directly referencing the color constants, right?